debezium/dbz#921 Optionally retain empty JSON array fields in outbox payload expansion (#7652)
debezium/debezium: documentation/modules/ROOT/pages/transformations/outbox-event-router.adoc

* `ignore` - Ignore the null value.
* `optional_bytes` - Keep the null value, and treat null as optional bytes of connect.

|[[outbox-event-router-property-table-json-payload-empty-array-behavior]]<<outbox-event-router-property-table-json-payload-empty-array-behavior, `table.json.payload.empty.array.behavior`>>
|`ignore`
|Table

> a|When enable JSON expansion property `table.expand.json.payload`, determines the behavior of a json payload that includes an empty array value on the outbox table. Possible settings are:

Reviewer: ```suggestion
a|When the JSON expansion property `table.expand.json.payload` is enabled, it determines how a JSON payload containing an empty array value is handled in the outbox table. 
Possible settings are:
```
